### ISSUE TITLE
Some typo of your paper

### ISSUE THREAD

**Issue by Keiku** [ChenhongyiYang/QueryDet-PyTorch#8]
> Our QueryDet achieves the performance with 43.2 AP. Specifically, with the aid of high-resolution features and our query supervision, our approach achieves 26.9 AP on small objects, which demonstrate the superiority of our method. Here, we use GIoU loss [44] and the modified Batch Normalization, as described in FCOS experiments, for further accuracy improvement.

The numbers 43.2 AP and 26.9 AP cannot be found in the Table 8.

<img src="https://user-images.githubusercontent.com/199433/120284289-116d0480-c2f7-11eb-8f23-ef2e0d6487d0.png" width="400px">

Also, in Figure 2, the QueryDet part is Query"N"et.

Just in case.

**Comment by ChenhongyiYang** [comment]
Thanks for reminds, we will correct it in the next version ~~ :-)
